### PR TITLE
Global cluster config : Replaced Keys with profile

### PR DESCRIPTION
**Pull request by SuperMohit**
## Description

Replaced Keys with Profile.
Test results:
Screenshot 2023-02-22 at 10 52 31 AM

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: cfn-resources/global-cluster-config/docs/README.md [hunk 1]
```diff
@@ -34,11 +34,13 @@ Properties:
 
 ## Properties
 
-#### ApiKeys
+#### Profile
+
+Profile used to provide credentials information, (a secret with the cfn/atlas/profile/{Profile}, is required), if not provided default is used
```
Comment (colm-quinn) [inline cfn-resources/global-cluster-config/docs/README.md:39]:
```suggestion
The profile is defined in AWS Secret manager. See [Secret Manager Profile setup](../../../examples/profile-secret.yaml).
```

File: cfn-resources/global-cluster-config/mongodb-atlas-globalclusterconfig.json [hunk 2]
```diff
@@ -41,43 +41,32 @@
...
     }
   },
-  "primaryIdentifier": ["/properties/ProjectId"],
+  "primaryIdentifier": ["/properties/ProjectId","/properties/Profile"],
   "properties": {
-    "ApiKeys": {
-      "$ref": "#/definitions/apiKeyDefinition"
+    "Profile": {
+      "type": "string",
+      "description": "Profile used to provide credentials information, (a secret with the cfn/atlas/profile/{Profile}, is required), if not provided default is used",
```
Comment (colm-quinn) [inline cfn-resources/global-cluster-config/mongodb-atlas-globalclusterconfig.json:68]:
```suggestion
      "description": "The profile is defined in AWS Secret manager. See [Secret Manager Profile setup](../../../examples/profile-secret.yaml).",
```

**Review by colm-quinn — APPROVED** [review]
Small improvements to use consistent text across all resources, otherwise LGTM!

**Review by colm-quinn — APPROVED** [review]
LGTM

**Comment by SuperMohit** [comment]
Screenshot 2023-02-24 at 5 06 17 PM

**Comment by govardhanpagidi** [comment]
@colm-quinn can you approve this PR?

**Review by colm-quinn — APPROVED** [review]

**Pull request merged**